Loop/Managers/Alerts/StoredAlert+CoreDataProperties.swift

```diff
@@ -21,7 +21,7 @@ extension StoredAlert {
     @NSManaged public var alertIdentifier: String
     @NSManaged public var backgroundContent: String?
     @NSManaged public var foregroundContent: String?
-    @NSManaged public var isCritical: Bool
+    @NSManaged public var interruptionLevel: Int16
```

Comment:
Consider making this a primitive and have a getter/setter to nicely map to `Alert.InterruptionLevel`.

Reply:
Actually, I decided to remove this altogether.  the (formerly) `isCritical` boolean was actually an aggregation of what is already stored in `foregroundContent` and `backgroundContent`.  As a `Bool` it sort of made sense to guess at it, but now that it is an enum I think it is duplicate at best and misleading at worst.